benchmarks/BenchmarkForall.cpp

```diff
@@ -69,6 +69,23 @@ static void benchmark_gpu_loop(benchmark::State& state) {
 // Register the function as a benchmark
 BENCHMARK(benchmark_gpu_loop)->Range(1, INT_MAX);
 
+static void benchmark_gpu_loop_streams(benchmark::State& state) {
```

Comment:
I'd like to see more benchmarks - especially one that uses threads to launch kernels in different streams.

Reply:
Where are the ones you used for your poster?

Reply:
these are the ones i used, but i can use the ones i made earlier with multiple kernels